src/MagnumPlugins/AssimpImporter/Test/Test.cpp

```diff
@@ -0,0 +1,257 @@
+/*
```

Comment:
Uh oh, seeems like I committed the vim swap file ~~accidentally~~ (on purpose, of course, so that everyone thinkgs I'm a vim-magician, hah :P )